crates/oxc_ast/src/ast/ts.rs

```diff
@@ -1595,6 +1595,7 @@ pub struct TSNamespaceExportDeclaration<'a> {
 pub struct TSInstantiationExpression<'a> {
     pub span: Span,
     pub expression: Expression<'a>,
+    #[estree(rename = "typeArguments")]
     pub type_parameters: Box<'a, TSTypeParameterInstantiation<'a>>,
```

Comment:
Let's rename this to `type_parameters` directly. The original implementation mixed these up.

Reply:
🙆🏻 I will update.